README.md

```diff
@@ -1,47 +1,123 @@
-section, or use it to frame the study. Good framing answers the question "Why am
-I learning this?".
-
-Be sure to include a recent [`LICENSE`](LICENSE) and Markdown linter
-configuration ([`.remarkrc`](.remarkrc)). Also, include an appropriate
-`.gitignore`; these are usually found in specific technology templates, for
-example [js-template](https://www.github.com/ga-wdi-boston/js-template).
+You've learned a lot about how to build a Node application over the last few
+weeks. Now let's 'go public' and share our apps with the world!
 
 ## Prerequisites
 
--   Topics with which developers should be familiar with.
--   Prerequisites are "just-in-time", so if I have a prerequisite that mentions
-    Sass, I would **not** need to include CSS as a prerequisite.
--   [Links to previous materials](https://www.github.com/ga-wdi-boston/example)
-    are often useful. Previous materials might include talks or other studies.
-
+- [Express Api](https://github.com/ga-wdi-boston/express-api)
+-   This guide assumes you have followed [these installation instructions](https://github.com/ga-wdi-boston/express-api-template#installation) unequivocally.
 ## Objectives
 
-By the end of this, developers should be able to:
+-   Create a Heroku app from the command line
+-   Push the latest code to Heroku
+-   Migrate the production database
+
+## Getting Set Up
+
+Before you can begin deploying your applications to Heroku, there are some
+things you'll need to do first.
+
+1.  **Create a Heroku account**, at [https://www.heroku.com](https://www.heroku.com).
+    You will be sent an activation email, so be sure to check your inbox so that
+    you can activate your account.
+1.  Install the Heroku Command Line Tools: run `brew install heroku`.
+1.  **Log into Heroku** by running `heroku login` from the console and providing
+    your Heroku credentials when asked. Once you log in, if you're prompted
+    to add these credentials to your keychain, say yes. *You will not be able*
+    *to see your password*
+1.  **sign up for mlab at mlab.com** / click link in verification email
+1.  **Credit card information needs to be submitted on heroku** - looks like as long as you don’t MASSIVELY populate your database, you’re credit card will not be charged
+1.  **Make sure you have heroku tool belt installed for MAC**
+
+## Deploying to Heroku
+
+Now you're set up to use Heroku. To deploy a new application
+to Heroku:
+
+
+-   [ ] Run `heroku create` in the command line in the root of your Express API to
+    create a new (blank) app on Heroku.
+-  [ ] Push your latest code to Heroku (`git push heroku master`)
+-  [ ] commit to your local master branch
+-  [ ] in terminal, run : `heroku create`
+-  [ ] in terminal run : `git remote -v` - make sure that heroku remote repo is pointing towards the newly created heroic url.   IF NOT, run this in terminal : `git remote set-url heroku <copy and paste the URL of your heroiku hosted app that you just created in step above>`
```

Comment:
I'm still not comfortable with this since if the heroku url isn't set there may be a more serious problem.

Also, spelling: `heroic`